spec/tasks/whitehall_importer_spec.rb

```diff
@@ -162,6 +183,16 @@
             ],
           },
         ],
+        "users" => [
+          {
+            "id" => 1,
+            "name" => "A Person",
+            "uid" => "36d5154e-d3b7-4e3e-aad8-32a50fc9430e",
+            "email" => "a-publisher@department.gov.uk",
+            "organisation_slug" => "a-government-department",
+            "organisation_content_id" => "01892f23-b069-43f5-8404-d082f8dffcb9",
+          },
+        ],
```

Comment:
This fixture data is repeated 3 times. Can we create a new shared `fixtures` file/folder that can be shared between all the tests?